x-pack/test/functional/apps/maps/layer_errors.js

```diff
@@ -31,7 +31,7 @@ export default function ({ getPageObjects }) {
       });
     });
 
-    describe('ESGeoGridSource with missing index pattern id', () => {
+    describe.skip('ESGeoGridSource with missing index pattern id', () => {
```

Comment:
Mind adding a comment as to why this is still skipped. Something like "TODO, skipped because ESGeoGridSource show no results icon instead of error icon."

Reply:
added the comment.